lua/entities/gmod_wire_expression2/core/custom/cfc_entity.lua

```diff
@@ -52,4 +52,21 @@ e2function void entity:ejectPodTo(vector pos)
 
     driver:ExitVehicle()
     driver:SetPos(clampedPos)
-end
\ No newline at end of file
+end
+
+
+-- Easy way to have some of the setPos benefits without mostly being able to be abused.
```

Comment:
The name of the function should probably explain what it does well enough. If there's any uncertainty, the E2 docs should clear it up. Users need to know what this function does, we should already know without needing a comment here.

Reply:
Removed the comment